src/syntax/types.rkt

```diff
@@ -134,5 +135,11 @@
                [vars (cons var (context-vars ctx))]
                [var-reprs (cons repr (context-var-reprs ctx))]))
 
+(define (context-append ctx var repr)
+  (struct-copy context
+               ctx
+               [vars (append (context-vars ctx) (list var))]
+               [var-reprs (append (context-var-reprs ctx) (list repr))]))
+
```

Comment:
Doesn't this do basically the same exact thing as `context-extend` except it puts the variable at the end? Why does it matter where the variable goes?